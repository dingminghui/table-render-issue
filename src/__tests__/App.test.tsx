import { render } from "@testing-library/react";
import App from "../App";

describe("App", () => {
  it("render <App /> correctly", () => {
    const result = render(<App />);
    expect(result.container).toMatchSnapshot();
  });
});
