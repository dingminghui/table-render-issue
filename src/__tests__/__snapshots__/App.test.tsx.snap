// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App > render <App /> correctly 1`] = `
<div>
  <h1>
    hi
  </h1>
</div>
`;
